runtime/v2/task/sandbox.proto

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
How about renaming "runtime/v2/task" as "runtime/v2/api" and renaning "runtime/v2/task/shim.proto" as "runtime/v2/api/task.proto"?

Reply:
I like this suggestion, however it produces huge diff, so I'll do it in a separate PR to ease review of this one.